flink-connectors/flink-connector-kafka-0.9/src/main/java/org/apache/flink/streaming/connectors/kafka/internal/KafkaConsumerThread.java

```diff
@@ -219,6 +209,15 @@ public void run() {
 					}
 				}
 
+				try {
+					newPartitions = unassignedPartitionsQueue.pollBatch();
+					if (newPartitions != null) {
+						reassignPartitions(newPartitions);
+					}
+				} catch (AbortedReassignmentException e) {
+					continue;
```

Comment:
Why is it necessary to continue here, instead of continuing with the rest of the loop?